.github/workflows/release.yaml

```diff
@@ -107,6 +112,11 @@ jobs:
 
           make clean test
 
+      - name: Verify analytics
```

Comment:
shouldn't that run even before the tests, so we don't lose time?

Reply:
Put it after, since it's less likely to fail than tests (thus not increasing CI time), but it works the other way too 😄 Can shift it up.